frontend/frontend.php

```diff
@@ -233,4 +235,32 @@ public function switch_blog( $new_blog_id, $prev_blog_id ) {
 			$this->load_strings_translations();
 		}
 	}
+
+	/**
+	 * Remove the customize admin bar on front-end when using a block theme.
+	 *
+	 * @since 3.2
+	 *
+	 * @return void
+	 */
+	public function remove_customize_admin_bar() {
+		if ( ! function_exists( 'wp_is_block_theme' ) || ! wp_is_block_theme() ) {
+			return;
+		}
+
+		global $wp_filter;
+		if ( empty( $wp_filter['customize_register'] ) ) {
+			return;
+		}
+
+		$customize_register_hooks = count( array_merge( ...array_values( $wp_filter['customize_register']->callbacks ) ) );
+		if ( $customize_register_hooks > 1 ) {
+			return;
+		}
```

Comment:
This part is common to `PLL_Admin_Base::remove_customize_submenu()`. You should be able to factorize it by creating a method in `PLL_Base`.